cou/apps/app.py

```diff
@@ -67,6 +72,9 @@ class Application:
     os_origin: str = ""
     channel: str = ""
     units: defaultdict[str, dict] = field(default_factory=lambda: defaultdict(dict))
+    current_os_release: str = ""
+    next_os_release: str = ""
+    os_versions: set = field(default_factory=set)
```

Comment:
why cache the set of versions? this could easily be grabbed from units

Reply:
It was to have this information in the object and be able to test like if we would have multiple OpenStack releases. However this is not necessary and adds some unnecessary lines of code. I've removed `os_versions` from `Application` and `Analysis` object.